Core/src/org/sleuthkit/autopsy/corecomponents/Installer.java

```diff
@@ -115,6 +119,16 @@ private void setOSXLookAndFeel() {
         });
     }
     
+    public static void setUIFont (javax.swing.plaf.FontUIResource f){
+    java.util.Enumeration keys = UIManager.getLookAndFeelDefaults().keys();
+    while (keys.hasMoreElements()) {
+      Object key = keys.nextElement();
+      Object value = UIManager.getLookAndFeelDefaults().get(key);
+      if (value instanceof javax.swing.plaf.FontUIResource)
+        UIManager.getLookAndFeelDefaults().put(key, f);
+      }
+    } 
+    
```

Comment:
We are not ready to commit to this approach. Please make a PR without setUIFont, but keep the code with setUIFont in another branch until we decide whether or not we want it.